src/apps/Altinn.AccessManagement/test/Bruno/Altinn.AccessManagement/Automatic Test Collection/AppsInstanceDelegation/DelegateMultipleRevokeAll/1.DelegatePerson1.bru

```diff
@@ -0,0 +1,109 @@
+meta {
+  name: 1.DelegatePerson1
+  type: http
+  seq: 1
+}
+
+post {
+  url: {{baseUrl}}/accessmanagement/api/v1/app/delegations/resource/{{resourceId}}/instance/{{revokeAllinstanceId}}
+  body: json
+  auth: none
+}
+
+headers {
+  PlatformAccessToken: {{platformAccessToken}}
+}
+
+body:json {
+  {
+    "from": {
+      "type": "urn:altinn:party:uuid",
+      "value": "{{fromUuid}}"
+    },
+    "to": {
+      "type": "urn:altinn:party:uuid",
+      "value": "{{toUuid}}"
+    },
+    "rights": [
+      {
+        "resource": [
+          {
+            "type": "urn:altinn:resource",
+            "value": "{{resourceId}}"
+          },
+          {
+            "type": "urn:altinn:task",
+            "value": "task_1"
+          }
+        ],
+        "action": {
+          "type": "urn:oasis:names:tc:xacml:1.0:action:action-id",
+          "value": "read"
+        }
+      },
+      {
+        "resource": [
+          {
+            "type": "urn:altinn:resource",
+            "value": "{{resourceId}}"
+          },
+          {
+            "type": "urn:altinn:task",
+            "value": "task_1"
+          }
+        ],
+        "action": {
+          "type": "urn:oasis:names:tc:xacml:1.0:action:action-id",
+          "value": "sign"
+        }
+      }
+    ]
+  }
+}
+
+script:pre-request {
+  const sharedtestdata = require(`./Testdata/sharedtestdata.json`);
+  const testdata = require(`./Testdata/instance-delegation/${bru.getEnvVar("tokenEnv")}.json`);
+  const resource = "app_" + sharedtestdata.serviceOwners.ttd.org + "_" + testdata.app;
+  
+  bru.setVar("resourceId", resource);
+  bru.setVar("revokeAllinstanceId", testdata.revokeallinstanceid);
+  bru.setVar("fromUuid", testdata.org1.partyuuid);
+  bru.setVar("toUuid", testdata.person1.partyuuid);
+  
+  var getTokenParameters = {
+      auth_org: sharedtestdata.serviceOwners.ttd.org,
+      auth_app: testdata.app,
+      auth_tokenType: sharedtestdata.authTokenType.platformAccess
+    };
+  
+  const token = await testTokenGenerator.getToken(getTokenParameters);
+  bru.setVar("platformAccessToken",  token);
+}
+
+tests {
+  test("post InstanceDelegation ReadSign", function() {
```

Comment:
Alle testene har nå det samme navnet: "post InstanceDelegation ReadSign"
som gjør det vanskelig å finne tilbake til hvilken request som evt. feiler.

Jeg har begynt å sette request navnet i en variabel i starten som jeg så bruker som test-navnet (i mangel på å finne en lett måte å bare kunne hentet .bru filnavnet direkte). Evt. kan man da også ha flere test funksjoner for samme request og bruke noa ala `requestName +"|" + testName`

F.eks:
```
// Should be the same as the .bru request file. Used as prefix in test name which also shows in test result in pipeline.
const requestName = "InstDelg_DelgChk_AppWithDelegableRights_Ok";
const data = res.getBody().data;

test(requestName, function() {
  expect(res.status).to.equal(200);
});
```